drivers/ethernet/eth_w5500.c

```diff
@@ -160,19 +160,18 @@ static int w5500_command(const struct device *dev, uint8_t cmd)
 	uint64_t end = sys_clock_timeout_end_calc(K_MSEC(100));
 
 	w5500_spi_write(dev, W5500_S0_CR, &cmd, 1);
-	do {
-		int64_t remaining = end - sys_clock_tick_get();
-
+  while(1) {
```

Comment:
Please check the alignment. See: https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html

Reply:
oops, I've used spaces instead of tabs.